src/components/layout/header/mobile-menu.ts

```
@@ -3,11 +3,14 @@
  * Handles mobile menu opening/closing functionality
  */
 
-export class MobileMenuManager {
+import type { Destroyable, EventCleanup } from "./types";
+
+export class MobileMenuManager implements Destroyable {
   private menu: HTMLElement | null = null;
   private toggleButton: HTMLElement | null = null;
   private overlay: HTMLElement | null = null;
   private isOpen: boolean = false;
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)